Tests/BacktraceBreadcrumbTests.swift

```diff
@@ -192,21 +188,10 @@ final class BacktraceBreadcrumbTests: QuickSpec {
                             }
                         }
 
-                        // Why the - 1?
-                        // Because one line is liable to get mangled by the wrapping - half will
-                        // be at the bottom and half at the top of the circular file.
-                        // Currently, we accept we lose this Breadcrumb in the UI - it will still be in the file
-                        // for manual inspection.
-                        let expectedNumberOfMatches = writeIndex - wrapIndex - 1
-                        expect(matches).to(beGreaterThanOrEqualTo(expectedNumberOfMatches),
```

Comment:
This check is important. I would consider adjusting it, not deleting it.

Reply:
This test is specifically there to make sure the file rolls over, so the number of expected matches are really important. Otherwise the entire test doesn't do anything the other (smaller) tests already do.